com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs

```diff
@@ -264,7 +264,7 @@ public void Dispose()
 
         public int callbackOrder { get { return 0; } }
 
-        void LogShaderVariants(ComputeShader shader, string kernelName, uint prevVariantsCount, uint currVariantsCount)
+        void LogShaderVariants(ComputeShader shader, string kernelName, uint prevVariantsCount, uint currVariantsCount, double stripTimeMs)
```

Comment:
Maybe we should start considering a struct as input of this method, or even doing it the otherway:
```csharp
class ShaderStripperReport
{
      bool GenerateLogString(out string log, out Exception error);
}
```

Reply:
This will come in another PR